Samples/WindowsML/Resources/SqueezeNetModelCatalog.json

```diff
@@ -0,0 +1,67 @@
+{
+  "base": "https://learn.microsoft.com/azure/ai-foundry/foundry-local/reference/reference-catalog-api",
+  "models": [
+    {
+      "alias": "squeezenet",
+      "name": "squeezenet",
+      "version": "1",
+      "modelType": "ONNX",
+      "publisher": "microsoft",
+      "executionProvider": "CPUExecutionProvider",
```

Comment:
Can you remind me how the implementation uses the EP info? Technically, I believe this model can run on the different NPUs, so would this be more accurately described via an array of those?

Reply:
We get EPs from ExecutionProviderCatalog::GetDefault().FindAllProviders() and filter models based on that. I can put array but for sample purposes I just kept cpu

Reply:
Thanks for the quick offline chat. A nice future addition would be to expand this to support other EP types.